modules/ROOT/partials/commands/inlinecss-cmds.adoc

```diff
@@ -0,0 +1,15 @@
+// mostly-blind guesswork.
```

Comment:
- [ ] remove unless someone can demonstrate it does exist :)

Reply:
Removed.